analysis_driver/tool_versioning.py

```diff
@@ -26,10 +27,6 @@ def select_version(self, version):
         self.version = version
         self.tools = {}
         self.tool_versions = {}
-
-        for k in cfg['tools']:
-            self.add_tool(k)
```

Comment:
Where will the tools be added now that this loop has been removed?

Reply:
See line 132.